src/routes/sessions/[id]/+page.svelte

```diff
@@ -207,17 +212,9 @@
 		promptCached = '';
 	}
 
-	function stopResponse(navigation?: BeforeNavigate) {
-		if (isWaitingForResponse) {
-			toast.warning(
-				"Can't stop completion until the first response from Ollama is received, please wait..."
-			);
-			navigation?.cancel();
-			return;
-		}
-
```

Comment:
We can now stop the request at any time so we no longer need this _hacky_ warning.